robber/expect.py

```diff
@@ -60,9 +60,14 @@ def registered(cls, name):
         else:
             return True
 
+    @property
+    def not_to(self):
+        self.is_negated = True
+        return self
+
     def __setup_chaining(self):
-        self.to   = self
-        self.be   = self
-        self.a    = self
-        self.an   = self
+        self.to = self
+        self.be = self
+        self.a = self
+        self.an = self
         self.have = self
```

Comment:
We should have a function something like `negate` to toggle the `is_negated` to handle the double/triple... negating as well, it will come in the next commit.